app/controllers/root_controller.rb

```diff
@@ -24,14 +24,15 @@ def govuk_component_docs
   end
 
   def govuk_available_locales
-    locale_files = Rails.root.join('app', 'views', 'locales', '*.yml')
+    locale_files = Rails.root.join("config", "locales", "*.yml")
     locales = Dir[locale_files].map { |file| File.basename(file, '.yml') }
     render :json => locales
   end
 
   def govuk_locales
     return error_404 unless params[:locale].match(/^[a-z]{2}(-[a-z0-9]{2,3})?$/)
```

Comment:
That regular expression will allow something like `pt-p1p`, which ISO-639 are we using and which ISO is that regex trying to accommodate?

https://en.wikipedia.org/wiki/ISO_639